packaging/macos/buildpackage.sh

```diff
@@ -278,10 +278,8 @@ if [ -n "${MACOS_DEVELOPER_CERTIFICATE_BASE64}" ] && [ -n "${MACOS_DEVELOPER_CER
 fi
 
 if [ -n "${MACOS_DEVELOPER_USERNAME}" ] && [ -n "${MACOS_DEVELOPER_PASSWORD}" ]; then
-	# Parallelize processing
```

Comment:
Perhaps add a comment about the notarisation issue

Reply:
Added a comment.